### PR TITLE
Language is required on Execute responses

### PR DESCRIPTION
**Pull request by jonescc**

### CODE REVIEW

**Comment by akashisama** [comment]
I'm not so sure about this. All the other configuration values are stored either in environment variables or application.properties but this one sits there as the one lonesome constant variable?

**Comment by jonescc** [comment]
Its a fair cop.

Modified to use a property as per other non instance specific config

File: wps-cloudformation-template.yaml [hunk 1]
```diff
@@ -108,6 +108,9 @@ Parameters:
   geonetworkLayerSearchField:
     Type: String
     Default: 'layer'
+  language:
```
Comment (bhasin85) [inline wps-cloudformation-template.yaml]: Where is this parameter getting used?

Comment (jonescc) [reply wps-cloudformation-template.yaml]: Whoops - I'll remove - originally started to add this as an environbment parameter but decide that wasn't appropriate

Comment (jonescc) [reply wps-cloudformation-template.yaml]: Removed

**Comment by jonescc** [comment]
@akashisama or @bhasin85 ?

**Pull request merged**